packages/cli/src/lingui-compile.ts

```diff
@@ -35,23 +44,25 @@ function command(config: LinguiConfig, options) {
   const doMerge = !!config.catalogsMergePath
   let mergedCatalogs = {}
 
-  console.error("Compiling message catalogs…")
+  console.log("Compiling message catalogs…")
```

Comment:
There was inconsistency in "log" and "error". Sometimes error used in place of log and vice-versa